### PR TITLE
Cumulus-298 Changed log filter to include all logs

### PR DESCRIPTION
**Pull request by kkelly51**
**Summary:** Changed log filter to include all logs

Addresses [CUMULUS-298: Not all CloudWatch log messages indexed by Elasticsearch](https://bugs.earthdata.nasa.gov/browse/CUMULUS-298)

## Changes

* Changed log filter to include all logs

## Test Plan
Things that should succeed before merging.

- [x] Integration tests
- [x] Update CHANGELOG

### CODE REVIEW

File: packages/api/es/search.js [hunk 1]
```diff
@@ -76,7 +76,7 @@ class BaseSearch {
     this.params = params;
     //log.debug('Generated params:', params, logDetails);
 
-    this.size = parseInt((params.limit) ? params.limit : 1, 10);
+    this.size = parseInt((params.limit) ? params.limit : 10, 10);
```
Comment (kkelly51) [inline packages/api/es/search.js]: We can discuss what we want the default limit to be. 1 log was too few as a single task could put out ~5 different log messages.

Comment (laurenfrederick) [reply packages/api/es/search.js]: I'd at least make this a const defined here so we can quickly change it or maybe an env variable.

Comment (scisco) [reply packages/api/es/search.js]: I think env variable is a great idea

File: packages/deployment/app/cloudformation.template.yml [hunk 2]
```diff
@@ -583,7 +583,7 @@ Resources:
           - log2elasticsearchLambdaFunction
           - Arn
       LogGroupName: '/aws/lambda/{{../stackName}}-{{@key}}'
-      FilterPattern: '{ $.name = "cumulus" }'
+      FilterPattern: ""
```
[the remark this answers is not in the document]
Comment (scisco) [reply packages/deployment/app/cloudformation.template.yml]: It seems that the empty one grabs all the logs, which is what we want

**Comment by scisco** [comment]
@laurenfrederick regarding your general comment, I think we actually do want to capture all the logs. There are going to be log messages that will not correspond to our log format. Any unexpected error message is in that category. The more logs we index on elasticsearch, the better we can figure out what went wrong in the system.

File: packages/deployment/app/cumulus_api.template.yml [hunk 3]
```diff
@@ -200,7 +200,7 @@ Resources:
       {{else}}
       Role: {{../parent.iams.lambdaProcessingRoleArn}}
     {{/if}}
-      Runtime: {{# if this.runtime}}{{this.runtime}}{{else}}nodejs6.10{{/if}}
+      Runtime: {{# if this.runtime}}{{this.runtime}}{{else}}nodejs8.10{{/if}}
```
Comment (scisco) [inline packages/deployment/app/cumulus_api.template.yml]: oh, thanks for catching this. I only updated the node version on the main template and not on the nested ones.

File: packages/deployment/app/cloudformation.template.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (kkelly51) [reply packages/deployment/app/cloudformation.template.yml]: Yes, it is a required parameter.

File: packages/api/es/search.js (hunk 1 above)
Comment (kkelly51) [reply packages/api/es/search.js]: I think this already exists. If we want to change the limit quickly or temporarily we can use the limit parameter.

**Pull request merged**